### PR TITLE
Close scalar markets

### PR DESCRIPTION
**Pull request by emaG3m**

### CODE REVIEW

File: contracts/Futarchy.sol [hunk 1]
```diff
@@ -19,7 +19,7 @@ contract Futarchy is AragonApp, IForwarder {
   event ExecuteDecision(uint decisionId);
   event BuyMarketPositions(address trader, uint decisionId, uint tradeTime, uint collateralAmount, uint[2] yesPurchaseAmounts, uint[2] noPurchaseAmounts, uint[2] yesCosts, uint[2] noCosts, uint[4] marginalPrices);
   event SellMarketPositions(address trader, uint decisionId, uint tradeTime, int[] yesMarketPositions, int[] noMarketPositions, uint yesCollateralReceived, uint noCollateralReceived, uint[4] marginalPrices);
-  event RedeemWinnings(address trader, uint decisionId, int winningIndex, uint winningsAmount);
+  event redeemWinningCollateralTokens(address trader, uint decisionId, int winningIndex, uint winningsAmount);
```
Comment (mikec) [inline contracts/Futarchy.sol]: should be capitalized

Comment (emaG3m) [reply contracts/Futarchy.sol]: git didn't check that I've since changed this?? (magic interactive rebases I guess)

File: contracts/Futarchy.sol [hunk 2]
```diff
@@ -148,61 +151,53 @@ contract Futarchy is AragonApp, IForwarder {
...
+      require(decisions[decisionId].decisionResolutionDate < now);
+      FutarchyOracle futarchyOracle = decisions[decisionId].futarchyOracle;
+      if(!futarchyOracle.categoricalEvent().isOutcomeSet()) {
+        if (!futarchyOracle.isOutcomeSet()) {
+          futarchyOracle.setOutcome();
+        }
+        CategoricalEvent(futarchyOracle.categoricalEvent()).setOutcome();
+      }
+
+      if(futarchyOracle.categoricalEvent().isOutcomeSet()) {
```
Comment (mikec) [inline contracts/Futarchy.sol]: we might be able to remove this .. I think this will always be true?

**Comment by mikec** [comment]
@emaG3m looooooks good 👍

**Pull request merged**